content/engineering/bruteforce-lock-and-unlock/index.md

```diff
@@ -0,0 +1,174 @@
+5. We can customize the predefined templates if needed.
+
+> The created app can be accessed using the link _[https://`<app-name>`.devhub.lrinternal.com/auth.aspx](https://`<app-name>`.devhub.lrinternal.com/auth.aspx)_ in the _implement_ section of the _Identity Experience Framework_ or can be accessed from the preview section.
+
+### Testing Brute force lockout
+Brute force lockout can be achieved by following these steps: 
+1. Register with a new user, and login with the correct user email and password. It is observed that the user is redirected to the profile page after authentication.
+<br>
+[![loginpage_with_data.png](assets/loginpage_with_data.png)](assets/loginpage_with_data.png)
+<br>
+[![successful_login.png](assets/successful_login.png)](assets/successful_login.png)
+<br>
+
+2. Now, logout and try to login with incorrect credentials.
+3. If the password is incorrect successively till the lockout threshold, the account gets locked. 
+4. Therefore, brute force lockout is achieved.
+<br>
+[![incorrect_pwd.png](assets/incorrect_pwd.png)](assets/incorrect_pwd.png)
+<br>
+
+> In the _Admin Console_, we can set the _brute force lockout threshold, lockout type, and suspend effective period_. 
+
+### Lockout types supported by LoginRadius
+_LoginRadius_ supports the following lockout types: 
+1. <u>Suspend:</u> 
+    - Suspends further login attempts after multiple failed tries for a certain amount of time, deterring automated attacks and enhancing system security by limiting access from suspicious sources.
+<br>
+2. <u>CAPTCHA:</u> 
+    - A security measure used to unlock a locked account on entering valid credentials by presenting a challenge to solve. 
+    - This challenge is often very easy for humans to solve but difficult for the bots.
+
+> _Refer [CAPTCHA](#captcha) in miscellaneous section to learn more_.
+
+<br>
+
+3. <u>Security Question:</u>
+    - A personalized query set up by the user to verify identity to unlock a locked account with valid credentials.
+<br>
+4. <u>Block:</u>
+    - Restricts login attempts from a specific source after multiple failed tries, enhancing security against unauthorized access.
+
+
+### Unlocking an account locked through Brute force lockout
+The locked user account can be unlocked in two ways, using: 
+1. Account Update API from the LoginRadius Account API collections.
+2. Auth Unlock Account by Access Token from the LoginRadius Authentication API collections.
+
+> For more understanding on _Auth Unlock Account_, refer [Auth Security Configuration](https://www.loginradius.com/docs/api/v2/admin-console/platform-security/auth-security-configuration/)
+
+#### Account Update API from the LoginRadius Account API collections
+- Calling the Account Update API with the provided enpoint, using the given method, providing the apisecret, and apikey formatting the given body will unlock the account.
+1.<u>Endpoint:</u> https://devapi.lrinternal.com/identity/v2/manage/account/{uid}
```

Comment:
@gayathrisuresh150501 We should mention the prod api details/domain.
cc: @ravitejag